src/lua/functions/lua_functions_loader.cpp

```diff
@@ -268,7 +268,7 @@ std::string LuaFunctionsLoader::getString(lua_State* L, int32_t arg) {
 	return std::string(c_str, len);
 }
 
-Position LuaFunctionsLoader::getPosition(lua_State* L, int32_t arg, int32_t& stackpos) {
+Position LuaFunctionsLoader::getLuaPosition(lua_State* L, int32_t arg, int32_t& stackpos) {
```

Comment:
I found the new name very weird. Maybe getPositionFromLua would be less confusing.

Reply:
Since is a method from LuaFunctionsLoader, why not just leave getPosition?

Reply:
> Since is a method from LuaFunctionsLoader, why not just leave getPosition?

Because with the changes I made, it started to conflict with the name of the getPosition function of the Player Lua class, since it inherits from the class LuaFunctionsLoader